packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.routing.js

```diff
@@ -0,0 +1,39 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.training.dashboard', {
+    url: '/dashboard',
+    component: 'pciProjectTrainingDashboardComponent',
+    resolve: {
+      breadcrumb: /* @ngInject */ () => null,
+      usage: /* @ngInject */ (OvhApiCloudProjectUsageCurrent, projectId) =>
+        OvhApiCloudProjectUsageCurrent.v6().get({ serviceName: projectId })
+          .$promise,
+      currencySymbol: /* @ngInject */ (OvhApiMe) =>
```

Comment:
`user` is probably already injected in some root state, we can probably get the currency code property from that instead of making another call